app/javascript/packs/widget.js

```diff
@@ -18,7 +17,7 @@ import { domPurifyConfig } from '../shared/helpers/HTMLSanitizer';
 const PhoneInput = () => import('../widget/components/Form/PhoneInput');
 
 Vue.use(VueI18n);
-Vue.use(Vuelidate);
+
```

Comment:
Test the widget functionalities like pre-chat form, form message (Chat Form)

Reply:
Email input message (Disable pre-chat form, the email input would appear on the first message)

Reply:
Tested this